src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -357,27 +336,12 @@ class BinaryCompactObject : public DomainCreator<3> {
   };
 
   struct UseProjectiveMap {
-    using group = EnvelopingCube;
+    using group = Envelope;
     using type = bool;
     static constexpr Options::String help = {
         "Use projective scaling on the frustal cloak."};
```

Comment:
can change `frustal cloak` to `frustums in the envelope` (`frustal cloak` is terminology that is used in a different `Domain` that does not become spherical, and at the time the BBH domain was written the two were similar enough to share this terminology). Since this language is not used anywhere else in this `Domain` it can be removed.